UnitTests/MPRoktTests.m

```diff
@@ -85,7 +85,8 @@ - (void)testSelectPlacementsExpandedWithValidParameters {
     NSString *viewName = @"testView";
     NSDictionary *attributes = @{@"key": @"value"};
     NSDictionary *finalAttributes = @{@"key": @"value", @"sandbox": @"true"};
-    NSDictionary *placements = @{@"placement": @"test"};
+    MPRoktEmbeddedView *exampleView = [[MPRoktEmbeddedView alloc] initWithFrame:CGRectZero];
```

Comment:
In Android it is called `RoktEmbeddedView`, should we align with the name on both the platforms ? cc: @Mansi-mParticle

Reply:
Namespacing isn't available in Objective C or Swift so [adding prefixes](https://google.github.io/styleguide/objcguide.html#prefixes) is the standard way of preventing naming collisions.